docs-2.0/3.ngql-guide/7.general-query-statements/6.show/17.show-sessions.md

```diff
@@ -0,0 +1,63 @@
+# SHOW SESSIONS
+
+`SHOW SESSIONS`语句显示所有会话信息，也可以指定会话ID进行查看。
+
+## 注意事项
+
+登录数据库时，会创建一个会话，操作结束执行`exit`退出登录时，会清空自身的会话信息。
```

Comment:
when using the console, if use the client API, call the release of the session means to sign out, and the service will delete the session info.